### ISSUE TITLE
What happened to the 'delete' for sectors and sector types?

### ISSUE THREAD

**Issue by matmaxgeds** [somaliaims/Somali-AIMSUI#433]
Am I not looking in the right place?

![Screenshot from 2019-12-17 15-26-24](https://user-images.githubusercontent.com/432153/71004058-d581b780-20e1-11ea-838d-38d63aadbae4.png)

I also don't see the one for deleting individual sectors any more?

**Comment by raashidahmad** [comment]
@matmaxgeds May be i misunderstood the requirements in this ticket and removed delete option on sectors. Please see the issue here:
https://github.com/somaliaims/Somali-AIMSUI/issues/286

**Comment by matmaxgeds** [comment]
@raashidahmad thanks, yes, this appears to be the issue. Hopefully clearer below:

Manual inputted sector type: user create and edit/delete
Manually inputted sector: user create and edit/delete
IATI sourced sector type: no user create, only system generated/edited, no user delete, but allow user name edit
IATI sourced sector: no user create, only system generated/edited, no user edit or delete

In other words, please put the delete buttons back for the user added sector types and sectors. We just don't want the management users deleting the IATI sourced sector types/sectors, but they can rename the sector types for IATI sourced sectors.

**Comment by raashidahmad** [comment]
@matmaxgeds Following fixes are made to this now:
1. Sector types from source - Can only be renamed.
2. Sector types user created - Can be edited/deleted
3. Sectors from source/IATI - Only created/edited by system
4. Sectors created by user - Can be edited/deleted
